ehsm_kms_service/function.js

```diff
@@ -264,8 +264,10 @@ const enroll_user_info = (action, DB, res, req) => {
   if (napi_res) {
     const { appid, apikey } = napi_res.result
     let cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
+    let sm_default_cmk_res = napi_result(cryptographic_apis.CreateKey, res, [0, 0])
     if (cmk_res) {

```

Comment:
Does the below change make sense?
if (sm_default_cmk_res && cmk_res)

Reply:
> Does the below change make sense? if (sm_default_cmk_res && cmk_res)

I think if you want to make cmk different from defaultcmk, you must write it separately

Reply:
the if (cmk_res) will be changed to if (sm_default_cmk_res && cmk_res)